content/writers-guide.md

```diff
@@ -10,10 +10,11 @@ title: Writer's Guide
 
 ## Article Structure
 
-1. Brief introduction - a paragraph or two so you get the basic idea. Tell what you are going to tell.
-2. Main content - tell what you promised to tell.
-3. Conclusion - tell what you told and recap the main points.
-4. References - link to related articles and external resources so people can read and learn more about the topic.
+1. Brief introduction - a paragraph or two so you get the basic idea about - the what and why. 
+2. Structure description for the "Main Content" - the how - how you will tell it in a brief paragraph (e.g. a new plugin or a webpack concept)
```

Comment:
Hi, by "how" I mean : how will you present the solution? There are many ways that something can be explained. A brief intro of that process is what, I believe, should be placed as the second point. The 1st point (1. Introduction)  should only focus on developing the motivation of the post, what is it and why we care. Before jumping to the full explanation, I think, a couple of lines of about how the author intend to explain his point will be great. So the reader know, in advance, what to expect as a general instruction of the main content.

Please let me know if I should elaborate more.